django_rq/management/commands/rqworker.py

```diff
@@ -77,7 +77,8 @@ def handle(self, *args, **options):
                 queues,
                 connection=queues[0].connection,
                 name=options['name'],
-                exception_handlers=get_exception_handlers() or None,
+                exception_handlers=get_exception_handlers(
```

Comment:
I don't think this is right. The function is defined as `get_exception_handlers(queue_name)` but you're passing in `queue.exception_handlers` here.